multiply_ui/server/context.py

```diff
@@ -1,11 +1,13 @@
 from typing import Optional, List
 
 from .model import Job
+from .controllers.controller import Controller
 
 
 class ServiceContext:
     def __init__(self):
         self._jobs = {}
+        self.controller = Controller()
```

Comment:
```suggestion
        self._controller = Controller()
```

Reply:
Actually there is no need for this. Controllers can be plain Python function with first parameter being the context passed from handlers.